MaxText/layers/gamma.py

```diff
@@ -0,0 +1,135 @@
+import functools
+import operator
+from typing import Sequence, Union, Callable, Tuple, Any
+from flax import linen as nn
+import common_types
+import jax
+import jax.numpy as jnp
+from jax.ad_checkpoint import checkpoint_name
+
+from layers import models
+from layers import normalizations
+from layers import attentions
+from layers import initializers
+from layers import embeddings
+from layers import linears
+from layers.linears import _convert_to_activation_function
+
+Embed = embeddings.Embed
```

Comment:
BTW I don't love this thing on lines 18 to 32. No reason for you to have to do rework because of it. But please don't replicate it in the future.